packages/app/src/app/overmind/namespaces/dashboard/utils.ts

```diff
@@ -22,7 +22,19 @@ export function sortByNameAscending(
   a: ProjectFragment,
   b: ProjectFragment
 ): number {
-  const repoA = a.repository.owner + '/' + a.repository.name;
-  const repoB = b.repository.owner + '/' + b.repository.name;
-  return repoA < repoB ? -1 : 1;
+  return a.repository.name.toLowerCase() < b.repository.name.toLowerCase()
```

Comment:
repositories are sorted just by repo name now, as it is more easy to spot one in a longer list with the new design